pyproject.toml

```diff
@@ -7,11 +7,13 @@ license = "MIT"
 readme = "README.md"
 packages = [{include = "professional_python_exercises_2_githubcli"}]
 
+[tool.poetry.scripts]
+githubcli = "professional_python_exercises_2_githubcli.cli.root:app"
```

Comment:
After installing the package the cli is available as `githubcli` or `poetry run githubcli` in the working dir.

Reply:
Appreciate the easy way of using it after install.